frontend/tests/features/cellCards/cellCards.test.ts

```diff
@@ -494,20 +523,18 @@ describe("Cell Guide", () => {
           .waitFor({ timeout: 5000 });
 
         const nodesLocator = `[data-testid^='${CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW_RECT_OR_CIRCLE_PREFIX_ID}']`;
-        const nodesBefore = await page.locator(nodesLocator).elementHandles();
+        const nodesBefore = await page.locator(nodesLocator).all();
         const numNodesBefore = nodesBefore.length;
 
-        const dummyChildLocator = `[data-testid='${CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW_RECT_OR_CIRCLE_PREFIX_ID}-dummy-child-CL:0000842__0-has-children-isTargetNode=false']`;
-        const dummyChild = (
-          await page.locator(dummyChildLocator).elementHandles()
-        )[0];
         // check that dummyChild is clickable
-        const isVisible = await dummyChild.isVisible();
-        expect(isVisible).toBe(true);
-        await dummyChild.click();
+        const dummyChildLocator = page.getByTestId(
+          `${CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW_RECT_OR_CIRCLE_PREFIX_ID}-dummy-child-CL:0000842__0-has-children-isTargetNode=false`
+        );
+        await dummyChildLocator.click();
```

Comment:
instead of manually using `elementHandles()[0]`, we can just use locator `getByTestId` and call `.click()`, which will automatically wait till the element is visible and clickable